src/components/ReviewOrdersDialog.tsx

```diff
@@ -195,6 +196,18 @@ function OrderContent({ order }: { order: DraftOrder }) {
         <OrderInformation title="Type" tooltipText={TOOLTIP_DESCRIPTIONS.TYPE}>
           {order.partiallyFillable ? "Partial fillable" : "Fill or Kill"}
         </OrderInformation>
+        <OrderInformation
+          title="Expiration date"
+          tooltipText={TOOLTIP_DESCRIPTIONS.VALID_TO}
+        >
+          {validToDate.toLocaleTimeString("en-GB", {
```

Comment:
why use `en-GB` here? Also I think we already have a function to format dates like this which we use in the table

Reply:
I think that this was the function that we used there